wireapp/wire-android
# feat: Add Anonymous Analytics User Profile - (WPB-8978)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8978" title="WPB-8978" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-8978</a>  [Android] Countly analytics ID and user properties
  </td></table>
  <br />
 






















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We had no way of setting a user profile to current logged in user

### Causes (Optional)

Not implemented.

### Solutions

Implement a way to observe current logged in account, changes on analytics settings and possible analytics identifier coming from other clients.

### Dependencies (Optional)

Needs releases with:

- [#] https://github.com/wireapp/kalium/pull/2887
- [#] https://github.com/wireapp/kalium/pull/2896

### Testing

#### Test Coverage (Optional)


#### How to Test

- Open App
- Login
- From `App Inspection` check you analytics identifier from Metadata table and verify against on our analytics tool.
- Same to be done when switching accounts
- - Login into same account on another platform (if android on same branch, better), already logged in user will receive a new tracking id from new logged in device and start using it
